app/javascript/elm/src/Main.elm

```diff
@@ -988,38 +987,34 @@ viewCrowdMapOptions isCrowdMapOn crowdMapResolution selectedSession tooltipIcon
         ]
 
 
-viewCrowdMapCheckBox : Bool -> Path -> Html Msg
-viewCrowdMapCheckBox isCrowdMapOn tooltipIcon =
+viewCrowdMapToggle : Bool -> Path -> Html Msg
+viewCrowdMapToggle isCrowdMapOn tooltipIcon =
     div [ class "filters__toggle-group" ]
-        [ input
-            [ id "checkbox-crowd-map"
-            , type_ "checkbox"
-            , checked isCrowdMapOn
-            , Events.onClick ToggleCrowdMap
-            ]
-            []
-        , label [ for "checkbox-crowd-map" ] [ text "Crowd Map" ]
+        [ label [] [ text "CrowdMap:" ]
+        , viewToggleButton "off" (not isCrowdMapOn) ToggleCrowdMap
+        , viewToggleButton "on" isCrowdMapOn ToggleCrowdMap
         , Tooltip.view Tooltip.crowdMap tooltipIcon
         ]
 
 
 viewCrowdMapSlider : String -> Html Msg
 viewCrowdMapSlider resolution =
     div [ id "crowd-map-slider" ]
-        [ p []
-            [ text "Resolution" ]
-        , div []
-            [ input
+        [ label [] [ text <| "grid cell size: " ++ (String.fromInt <| 51 - (Maybe.withDefault 20 <| String.toInt resolution)) ]
+
+        -- size 40 to 1 maps to resolution 11 to 50
+        , div [ class "crowd-map-slider-container" ]
+            [ span [ class "minus" ] [ text "-" ]
+            , input
                 [ class "crowd-map-slider"
                 , onChange (String.toInt >> Maybe.withDefault 25 >> UpdateCrowdMapResolution)
                 , value resolution
                 , max "50"
-                , min "10"
+                , min "11"
                 , type_ "range"
                 ]
                 []
-            , span []
-                [ text resolution ]
+            , span [ class "plus" ] [ text "+" ]
```

Comment:
Would be nice if these were buttons? When I opened the page I instinctively tried to click them 😉

Reply:
yeah, I know :) for now they're just indicators

Reply:
thx for being a test user and confirming it :D I added a ticket for that now